src/include/sof/audio/component_ext.h

```diff
@@ -345,7 +345,9 @@ static inline int comp_get_requested_state(int cmd)
 	case COMP_TRIGGER_RESET:
 		state = COMP_STATE_READY;
 		break;
-	default:
+	case COMP_TRIGGER_PRE_START:
+	case COMP_TRIGGER_PRE_RELEASE:
+		state = COMP_STATE_PRE_ACTIVE;
```

Comment:
Dont we need the POST_states here too (and in other places) ?

Reply:
@lgirdwood I'd very much not want to add any code for ephemeral theoretical presumed POST_* things. I'm actually even against adding those command macros in the first place. I really don't find "symmetry" a sufficient argument for adding code when we have no intention whatsoever to use them. Or do we? I'd much rather remove any POST_* stuff from this PR and, if absolutely desired, it can be added later.

Reply:
ok, as long as the POST is defined as macros and covered by the ```default:``` case then we are good. I would comment next to the POST macro that POST is not fully implemented.